upstream-community-operators/kubevirt/kubevirtoperator.0.17.0-alpha.0.csv.yaml

```
@@ -0,0 +1,566 @@
+        - serviceAccountName: kubevirt-operator
+          rules:
+            - apiGroups:
+              - kubevirt.io
+              resources:
+              - kubevirts
+              verbs:
+              - get
+              - list
+              - watch
+              - patch
+              - update
+              - patch
+            - apiGroups:
+              - ""
+              resources:
+              - serviceaccounts
+              - services
+              - endpoints
+              verbs:
+              - get
+              - list
+              - watch
+              - create
+              - update
+              - delete
+              - patch
+            - apiGroups:
+              - batch
+              resources:
+              - jobs
+              verbs:
+              - get
+              - list
+              - watch
+              - create
+              - delete
+              - patch
+            - apiGroups:
+              - apps
+              resources:
+              - deployments
+              - daemonsets
+              verbs:
+              - get
+              - list
+              - watch
+              - create
+              - delete
+              - patch
+            - apiGroups:
+              - rbac.authorization.k8s.io
+              resources:
+              - clusterroles
+              - clusterrolebindings
+              - roles
+              - rolebindings
+              verbs:
+              - get
+              - list
+              - watch
+              - create
+              - delete
+              - patch
+              - update
+            - apiGroups:
+              - apiextensions.k8s.io
+              resources:
+              - customresourcedefinitions
+              verbs:
+              - get
+              - list
+              - watch
+              - create
+              - delete
+              - patch
+            - apiGroups:
+              - security.openshift.io
```

This operator seems to depend on a few OpenShift apis, yet it's being added to the `upstream-community-operators` directory which is meant for operators that can run on vanilla kubernetes
Can this operator run on vanilla kubernetes with the lack of SCCs?

---

I have tested it in vanial K8s  and works fine, as per Kubevirt it works on K8s and OKD., But will check with Kubevirt team.

---

Hi, yes, we are testing the operator in our CI on k8s and openshift clusters. Seems that non existent apigroups in rules are silently ignored :slightly_smiling_face:

---

Tested it works.